lib/stale.js

```diff
@@ -33,6 +40,12 @@ module.exports = class Stale {
     const days = this.config.days || this.config.daysUntilClose
     return this.search(days, query)
   }
+  
+  getLockableIssues () {
+    const query = `label:"${this.config.staleLabel}"`
+    const days = this.config.days || this.config.daysUntilLock
+    return this.search(days, query)
```

Comment:
The current search method adds `is:open` and `updated:<${timestamp}`. So I think this will need refactored so activity on the issue doesn't delay the locking.

Reply:
I've added a third param to search for `is:closed` instead of `is:open`, but that doesn't address your above comment. Shouldn't activity prevent it from locking? If it is considered stale, it's unlikely to get more attention (unless people still think it should). I think the normal use case for locking issues is for one that's been closed and around a really long time without activity.